src/styles/index.scss

```diff
@@ -1,3 +1,16 @@
+@import './utils/variables';
+@import './blocks/card';
+@import './blocks/stars';
+
+html {
+  margin: 0;
+  padding: 0;
+  font-family: $font-family-primary;
+  font-weight: $font-weight-regular;
```

Comment:
Ensure that the variables `$font-family-primary` and `$font-weight-regular` are defined in the 'variables.scss' file. If they are not defined, the styles will not be applied correctly.